Source/WebCore/html/LinkIconCollector.h

```diff
@@ -35,15 +35,15 @@ enum class LinkIconType : uint8_t;
 
 class LinkIconCollector {
 public:
-    explicit LinkIconCollector(Document& document)
+    explicit LinkIconCollector(Ref<Document> document)
```

Comment:
We do not pass smart pointers by value due to ref-counting churn. Please use a r-value reference and `WTFMove()` below.